typespec-tests/src/main/java/com/cadl/multicontenttypes/implementation/MultipleContentTypesOnRequestsImpl.java

```diff
@@ -176,9 +175,8 @@ public Mono<Response<Void>> uploadBytesWithSingleBodyTypeForMultiContentTypesWit
      * BinaryData
      * }</pre>
      * 
-     * @param contentType The contentType parameter. Allowed values: "application/octet-stream", "image/jpeg",
-     * "image/png".
-     * @param data The data parameter.
+     * @param contentType The contentType parameter.
```

Comment:
This change may also due to the difference of logic in `CustomClientParameterMapper`